analyzer/src/main/kotlin/managers/utils/NuGetSupport.kt

```diff
@@ -62,20 +66,27 @@ import org.ossreviewtoolkit.utils.await
 import org.ossreviewtoolkit.utils.collectMessagesAsString
 import org.ossreviewtoolkit.utils.log
 import org.ossreviewtoolkit.utils.logOnce
+import org.ossreviewtoolkit.utils.searchUpwardsForFile
 
-// See https://docs.microsoft.com/en-us/nuget/api/overview.
-private const val SERVICE_INDEX_URL = "https://api.nuget.org/v3/index.json"
-
+private const val DEFAULT_SERVICE_INDEX_URL = "https://api.nuget.org/v3/index.json"
 private const val REGISTRATIONS_BASE_URL_TYPE = "RegistrationsBaseUrl/3.6.0"
 
-class NuGetSupport {
+class NuGetSupport(vararg serviceIndexUrls: String = arrayOf(DEFAULT_SERVICE_INDEX_URL)) {
```

Comment:
Making this a list like

    class NuGetSupport(serviceIndexUrls: List<String> = listOf(DEFAULT_SERVICE_INDEX_URL)) {

makes the code below easier.